notebooks/vertex_matching_engine/solutions/matching_engine_query.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>Add dependency to https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/requirements.txt</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>